app/controllers/playlists_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PlaylistsController < ApplicationController
+
+  def index
+    @playlists = Playlist.all
+  end
+
+  def show
+    @playlist = Playlist.find(params[:id])
+  end
+
+  def new
+    @playlist = Playlist.new
+  end
+
+  def create
+    @playlist = Playlist.new(playlist_params)
```

Comment:
```suggestion
    @playlist = Playlist.new(playlist_params)
    
```

c'est plus lisible avec un espace ici